src/index.json

```diff
@@ -1827,53 +1827,97 @@
                 "sha256Digest": "6f4789fff8581a52e13bcf8f13fdc16b47662fac69e1fc3ea49280de1a8e65bc"
             }
         ],
-        "maintenance": [
-            {
-                "downloadUrl": "https://mrpcliextension.blob.core.windows.net/cliextension/maintenance-1.0.0-py2.py3-none-any.whl",
-                "filename": "maintenance-1.0.0-py2.py3-none-any.whl",
-                "metadata": {
-                    "azext.isPreview": true,
-                    "azext.minCliCoreVersion": "2.0.47",
-                    "classifiers": [
-                        "Development Status :: 4 - Beta",
-                        "Intended Audience :: Developers",
-                        "Intended Audience :: System Administrators",
-                        "Programming Language :: Python",
-                        "Programming Language :: Python :: 2",
-                        "Programming Language :: Python :: 2.7",
-                        "Programming Language :: Python :: 3",
-                        "Programming Language :: Python :: 3.4",
-                        "Programming Language :: Python :: 3.5",
-                        "Programming Language :: Python :: 3.6",
-                        "License :: OSI Approved :: MIT License"
-                    ],
-                    "extensions": {
-                        "python.details": {
-                            "contacts": [
-                                {
-                                    "email": "abkmr@microsoft.com",
-                                    "name": "Abhishek Kumar",
-                                    "role": "author"
-                                }
-                            ],
-                            "document_names": {
-                                "description": "DESCRIPTION.rst"
-                            },
-                            "project_urls": {
-                                "Home": "https://github.com/Azure/azure-cli-extensions"
-                            }
-                        }
-                    },
-                    "generator": "bdist_wheel (0.30.0)",
-                    "license": "MIT",
-                    "metadata_version": "2.0",
-                    "name": "maintenance",
-                    "summary": "Support for Azure maintenance management preview.",
-                    "version": "1.0.0"
+      "maintenance": [
+        {
+          "downloadUrl": "https://mrpcliextension.blob.core.windows.net/cliextension/maintenance-1.0.0-py2.py3-none-any.whl",
```

Comment:
Keep one item is enough. Please delete the old one.

Reply:
Sure. Deleted the older version